config/application.rb

```diff
@@ -3,7 +3,8 @@
 require 'rails/all'
 
 ActiveSupport.on_load(:active_record) do
-  unless $last_initializer_loaded
+  # Some rails tasks like db:create may load database before envoronment
+  if $environment_loaded && !$last_initializer_loaded
```

Comment:
Seems like this is an existing problem based off of the warning message below, but is there some sort of race condition you are seeing on boot?

Reply:
We had an issue with gems and initializers loading ActiveRecord before all initializers have run thus some configuration options not being respected. This is a safety hook to warn against this.